eng/mgmt/mgmtmetadata/storage_resource-manager.txt

```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/storage/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp.output-folder=C:\code\azure-sdk-for-net\sdk\storage\Microsoft.Azure.Management.Storage\src\Generated
-2020-04-22 10:53:21 UTC
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/storage/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp.output-folder=C:\code\srpsdk\sdk\storage\Microsoft.Azure.Management.Storage\src\Generated
+2020-05-25 08:38:19 UTC
 Azure-rest-api-specs repository information
 GitHub fork: Azure
 Branch:      master
-Commit:      d365bebac567c72ec1ab24bb3b7dbe2ae7caab1d
+Commit:      d413a0acb59a22f4d551ca598d96794715de8e40
 AutoRest information
 Requested version: v2
-Bootstrapper version:    autorest@2.0.4413
+Bootstrapper version:    autorest@2.0.4283
```

Comment:
```suggestion
Bootstrapper version:    autorest@2.0.4283
```
Hi @blueww I notice this is generated by an older version of autorest, however the generated code looks good, so I'll just merge the PR.

A `autorest --reset` should help update the version.